consensus/hotstuff_leader.go

```diff
@@ -145,19 +159,26 @@ func (handler *HotstuffLeaderMessageHandler) HandlePrecommitMessage(m *consensus
 	handler.emitTelemetryEvent(m, msg)
 
 	if err := handler.anteHandle(m, msg); err != nil {
-		m.nodeLogError(typesCons.ErrHotstuffValidation.Error(), err)
+		m.logger.Error().Err(err).Msg(typesCons.ErrHotstuffValidation.Error())
 		return
 	}
 
 	if err := m.didReceiveEnoughMessageForStep(PreCommit); err != nil {
-		m.nodeLog(typesCons.OptimisticVoteCountWaiting(PreCommit, err.Error()))
+		m.logger.Info().Msg(typesCons.OptimisticVoteCountWaiting(PreCommit, err.Error()))
 		return
 	}
-	m.nodeLog(typesCons.OptimisticVoteCountPassed(m.height, PreCommit, m.round))
+
+	m.logger.Info().Fields(
+		map[string]any{
+			"height": m.height,
+			"round":  m.round,
+			"step":   Prepare,
```

Comment:
Not sure If I understand the comment

Reply:
s/Prepare/PreCommit

This is inside the `HandlePrecommitMessage` function, so I think we should be logging `"step": PreCommit` instead of `"step": Prepare` which we do  in the `HandlePrepareFunction`